tests/src/test/spark350emr/org/apache/spark/sql/rapids/execution/GpuSubqueryBroadcastExecSuite.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Comment:
This appears to be a brand new file.
```suggestion
 * Copyright (c) 2024, NVIDIA CORPORATION.
```

Reply:
Will fix.